src/content/_posts/2020-10-08-working-with-the-community-and-launch-of-the-new-design.md

```diff
@@ -0,0 +1,85 @@
+
+Also I tried to make the **hard** things *easier* by throwing out some [open source stuff](https://github.com/biohazard999?tab=repositories&q=&type=source&language=).  
+To mention a few with more traction:
+
+* [DXNugetPackageBuilder](https://github.com/biohazard999/DXNugetPackageBuilder)
+* [FluentModelBuilder](https://github.com/biohazard999/Para.FluentModelBuilder)
+* [DXImageAssemblyImporter](https://github.com/biohazard999/DXImageAssemblyImporter)
+* [WindowsIntegration](https://github.com/biohazard999/Xpand.ExpressApp.Win.Para.WindowsIntegration) (this was the first one)
+
+Some of them are obsolete now (`DXNugetPackageBuilder` is not longer needed, but for legacy apps it's still valuable).
+
+I put a lot of effort and time into this in the past and I am always happy if I put something out there that helped *somebody else* and I will continue to do so.
+
+This is the **giving** back to the community part.
+But a community is not only about *giving* it's also about **taking**.
+
+I do it every day. Using open source, reading blog-posts, articles, docs, samples, reaching out to people helping me to achieve **my goal**.
+
+People are investing hundreds of thousands of hours into problem solving online.
+
+* Write a blog post about a problem so I have a space to look at if I ever encounter this again
+* Write a tutorial / in depth for reference
+* Write open source software
+* Contribute by
+  * Comment
+  * Fix spelling
+  * Provide documentation
+  * Testing
+  * Filing bugs/issues/ideas
+* Provide funding
+
+The last point is interesting, cause we all know that we need money for living, but let me focus know on parts that I don't think are part of an open community:
+
+* Spam
+* Being rude
+* Try to get support for free
+* Be upset cause somebody of the community does not respond quick enough / at all
+
+The first 2 points are easy to explain. Nobody likes spam and nobody likes rude people. Like all people we have opinions and can disagree on topics, but we are professionals and should act like ones. We can discuss problems like grown ups and, of course, nobody can like everybody, but can talk on a professional level.
+
+The second 2 points are more interesting.  
+We have a lot of places to discuss today. We have [user groups](), [live community standups](), we have [online forums](), [Gitter Chat rooms](), [Facebook groups](), [LinkedIn](), [Github](), etc. We **love** to talk about stuff that gets us excited, we share ideas, solutions and ask questions.  **BUT** I don't think its a place to get free support. It's a chance to *speak* with experts *for free*. That does not mean you shouldn't ask questions! It only means, we only can open doors. You need to go through yourself.  
+We are not your search engine. We are the search engine for a problem you googled for hours and can't find an answer. Probably cause you asked the *wrong question*.  
```

Comment:
```suggestion
We are not a search engine. We are the search engine for a problem you googled for hours and couldn't find an answer to. Perhaps because you asked the *wrong question*. 
```